funding/manager.go

```diff
@@ -2786,7 +2786,7 @@ func (f *Manager) annAfterSixConfs(completeChan *channeldb.OpenChannel,
 		}
 
 		log.Debugf("Channel with ChannelPoint(%v), short_chan_id=%v "+
-			"announced", &fundingPoint, shortChanID)
+			"sent to gossip", &fundingPoint, shortChanID)
```

Comment:
can probably remove

Reply:
Do you mean the log or the new change?

Reply:
I just meant the new change and maybe having a separate pull that changes the logs?

Reply:
Now lives in its own commit.

Reply:
maybe "sent to gossiper" instead?

Reply:
Done!